gensim/utils.py

```diff
@@ -1348,7 +1348,7 @@ def smart_extension(fname, ext):
 
 
 def pickle(obj, fname, protocol=2):
-    """Pickle object `obj` to file `fname`, using smart_open so that `fname` can be on S3, HDFS, compressed etc.
+    """Pickle object `obj` to file `fname`, using open so that `fname` can be on S3, HDFS, compressed etc.
```

Comment:
Wrong replacement.

Reply:
I changed the usage in documented function, do you want me to leave the comments with `smart_open`? same for the `unpickle` comment below?

Reply:
Yeah, the replacements need to make sense.

`Pickle X using open so that it can be on S3, HDFS, compressed etc.` is a totally confusing comment. `open` cannot do anything about S3, HDFS, etc (the `smart_open` library can).

Reply:
ok, no problem, reverting those comments to original

Reply:
@piskvorky fixed